packages/scss/src/components/tag/component.scss

```diff
@@ -15,7 +15,8 @@
 	display: inline-flex;
 	align-items: center;
 	text-align: center;
-	vertical-align: baseline;
+	vertical-align: top;
+	margin-block-start: var(--pr-t-spacings-25);
```

Comment:
Shouldn't this be only in context of Table ? 🤔

Reply:
It's needed all the time. In the future, the one in the table will disappear.

Reply:
Here's a before/after.
The main objective was to avoid alignment shifts in red.

![Capture d’écran 2025-03-27 à 11 35 44](https://github.com/user-attachments/assets/a03ce726-50a5-4b43-941f-65737e419bca)